andela/ah-kgl-avengers-frontend
# #166978192 Fix login and sign up using social auth

### What does this PR do?
Fix the login and sign up using social auth

### Description of Task to be completed?
- User should be able to log in via Facebook
- User should be able to log in via google
### How should this be manually tested?
- Clone the repository locally with `git clone https://github.com/andela/ah-kgl-avengers-frontend.git` command.
- Check out to this branch by running `git checkout ch-Fix-login-and-sign-up-using-social-auth-166978192 `.
- Install dependencies by running `npm install`.
- Update the .env file according to the .env.sample file
- Start the server with `npm start` and put this url `http://localhost:8000/` in the address bar.
- Login or Sign up with either Facebook or Google
### Any background context you want to provide?
This ticket also fixes the responsiveness of the social login buttons.
### What are the relevant pivotal tracker stories?
[#166978192 Fix the login and sign up using social auth](https://www.pivotaltracker.com/story/show/166978192)
### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/33352484/60554605-7ea54300-9d38-11e9-84aa-8fc43496888d.png)

### Questions:
N/A
